src/promptflow/promptflow/tracing/_utils.py

```diff
@@ -59,18 +58,23 @@ def serialize(value: object, remove_null: bool = False, serialization_funcs: Dic
 
 
 def get_input_names_for_prompt_template(template_str):
-    input_names = []
-    env = Environment()
-    template = env.parse(template_str)
-    input_names.extend(sorted(meta.find_undeclared_variables(template), key=lambda x: template_str.find(x)))
+    try:
```

Comment:
Add more documents about the try logic here

Reply:
Maybe we can try except then directly return a empty list if ImportError.
So we don't need to update the other codes.

Reply:
Added.